### PR TITLE
feat: add badge to budget card & fix query filter of budget analyze 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [x] Not that difficult
- [ ] Approved feature branch merge to master

### Description
스크린샷 2023-12-13 오전 10 33 34

### CODE REVIEW

**Review by skdud4659 — APPROVED** [review]
LGTM~

File: apps/web/src/schema/cost-analysis/budget-usage/api-verbs/analyze.ts [hunk 1]
```diff
@@ -5,12 +5,13 @@ export interface BudgetUsageAnalyzeResult {
...
     name?: string;
     project_id?: string;
-    project_group_id?: string;
+    workspace_id?: string;
     data_source_id?: string;
     provider_filter?: {
         state?: string;
         providers?: string[];
     };
+    permission_group: 'WORKSPACE' | 'DOMAIN';
```
Comment (sulmoJ) [inline apps/web/src/schema/cost-analysis/budget-usage/api-verbs/analyze.ts:14]: nitpick: We have [PermissionGroup](https://github.com/cloudforet-io/console/blob/aaa2b1487668fc0b5e517e782edd188561813010/apps/web/src/schema/identity/role-binding/type.ts#L1C1-L1C66) type!!

Comment (sulmoJ) [reply apps/web/src/schema/cost-analysis/budget-usage/api-verbs/analyze.ts:14]:
```suggestion
Pick<PermissionGroup, 'WORKSPACE' | 'DOMAIN'>
```

**Review by sulmoJ — APPROVED** [review]
LGTM~~

**Pull request merged**